### PR TITLE
fix(cli): the --all-scripts flag should skip scripts that don't call the webpack cli

### LINKED ISSUE #111: Expected behaviour of --all-scripts flag
## Issue

Given the following:

```
  "scripts": {
    "development": "webpack --config config/webpack.config.development.js",
    "production": "webpack --config config/webpack.config.production.js",
    "validate": "webpack-validator --all-scripts"
  }
```

Running:

`> npm run validate`

Produces the following error:

```

Validating development
Reading: config/webpack.config.development.js
[webpack-validator] Config is valid.
[webpack-validator] Config is valid.
development is valid

Validating production
Reading: config/webpack.config.production.js
[webpack-validator] Config is valid.
[webpack-validator] Config is valid.
production is valid

Validating validate
Reading: webpack.config.js
module.js:442
    throw err;
    ^

Error: Cannot find module 'D:\Dev\webpack-2-spike\webpack.config.js'
```

## Expected
I expected webpack-validator to only execute against the `development` and `production` scripts as they have the `--config` flag.

## Investigation

The issue is that it also parses the `validate` script, which doesn't have a `--config` flag and in this case, the code defaults to `webpack.config.js` which doesn't exist.

See https://github.com/js-dxtools/webpack-validator/blob/b52bf8c298c7da0c9bb4ff493752f1ce29c23bda/src/bin/script-parsers.js#L11

## Question

Is this the expected behavior, or should we ignore scripts that don't point to config files?

### PR DESCRIPTION
**Pull request by michael-wolfenden**
Closes #111

### CODE REVIEW

File: src/bin/script-parsers.test.js [hunk 1]
```diff
@@ -12,6 +12,47 @@ describe('webpack config name parser', () => {
...
+      'npm-run-all --parallel validate-webpack:* lint test --serial check-coverage',
+      'webpack-validator webpack.config.js --env.dev',
+      'webpack-validator webpack.config.js --env.prod',
+      'rimraf dist',
+      'cpy src/index.html src/favicon.ico dist',
+      'npm run clean-dist && npm run copy-files',
+      'npm run clean-and-copy',
+      'webpack-dev-server --env.dev --content-base dist',
+      'npm run clean-and-copy',
+      'npm run clean-and-copy',
```
Comment (kentcdodds) [inline src/bin/script-parsers.test.js]: Is there any reason to have multiple of the same script in this array?

Comment (michael-wolfenden) [inline src/bin/script-parsers.test.js]: My bad, I had just copied the scripts from:
https://github.com/kentcdodds/es6-todomvc/blob/c7838ebff426c15bdc16344863bacda11e175508/package.json

I didn't realise that `prebuild` and `prebuild:prod` were the same

Comment (kentcdodds) [inline src/bin/script-parsers.test.js]: haha. I thought those scripts looked familiar ;-)

**Comment by kentcdodds** [comment]
LGTM. But I should say that I don't use this feature, so someone else should probably give this a look over :-)

**Comment by jonathanewerner** [comment]
Pinging @bebraw for this. 👋 :)

**Comment by bebraw** [comment]
Looks good to me.

**Pull request merged**